conda.el

```diff
@@ -281,20 +295,22 @@ It's platform specific in that it uses the platform's native path separator."
 (defun conda-env-deactivate ()
   "Deactivate the current conda env."
   (interactive)
-  (run-hooks 'conda-predeactivate-hook)
-  (conda--set-python-shell-virtualenv-var nil)
-  (setq exec-path (conda-env-stripped-path exec-path))
-  (setenv "PATH" (s-join path-separator
-                  (conda-env-stripped-path
-                   (s-split path-separator (getenv "PATH")))))
-  (setenv "VIRTUAL_ENV" nil)
-  (setenv "CONDA_PREFIX" nil)
-  (setq conda-env-current-name nil)
-  (setq eshell-path-env (getenv "PATH"))
-  (conda--set-system-gud-pdb-command-name)
-  (run-hooks 'conda-postdeactivate-hook)
-  (when (called-interactively-p 'interactive)
-    (message "conda env deactivated")))
+  (when (bound-and-true-p conda-env-current-name)
+    (run-hooks 'conda-predeactivate-hook)
+    (conda--set-python-shell-virtualenv-var nil)
+    (setq exec-path (conda-env-stripped-path exec-path))
+    (message "exec-path: %s" exec-path)
```

Comment:
Is this `(message)` call still necessary?

Reply:
A forgotton debug output :)